quelpa.el

```diff
@@ -548,9 +555,9 @@ position."
 ;;; Run Process
 
 (defun quelpa-build--run-process (dir command &rest args)
-  "In DIR (or `default-directory' if unset) run COMMAND with ARGS.
+  "In DIR (or symbol `temporary-file-directory' if unset) run COMMAND with ARGS.
```

Comment:
Why `symbol` here?  I haven't seen a variable prefixed that way in a docstring before.

Reply:
Sorry, just to keep the line under 80 characters. I can change this back to variable